snippets/off.md

```diff
@@ -0,0 +1,16 @@
+### off
+
+Removes an event listener from an element.
+
+Use `EventTarget.removeEventListener()` to remove an event listener from an element.
+
```

Comment:
Add *Omit `opts` to use a default value of `false`* and explain what difference it makes.